executor/analyze.go

```diff
@@ -435,3 +444,73 @@ func (e *AnalyzeColumnsExec) buildStats() (hists []*statistics.Histogram, cms []
 	}
 	return hists, cms, nil
 }
+
+func analyzeFastExec(exec *AnalyzeFastExec) []statistics.AnalyzeResult {
+	hists, cms, err := exec.buildStats()
+	if err != nil {
+		return []statistics.AnalyzeResult{{Err: err}}
+	}
+	var results []statistics.AnalyzeResult
+	hasIdxInfo := len(exec.idxsInfo)
+	hasPKInfo := 0
+	if exec.pkInfo != nil {
+		hasPKInfo = 1
+	}
+	if hasIdxInfo > 0 {
+		for i := hasPKInfo + len(exec.colsInfo); i < len(hists); i++ {
+			idxResult := statistics.AnalyzeResult{
+				PhysicalTableID: exec.PhysicalTableID,
+				Hist:            []*statistics.Histogram{hists[i]},
+				Cms:             []*statistics.CMSketch{cms[i]},
+				IsIndex:         1,
+			}
+			if hists[i].Len() > 0 {
+				idxResult.Count += hists[i].Buckets[hists[i].Len()-1].Count
```

Comment:
So the index needs `not null count` while the column needs `total count`?

Reply:
Yes.

Reply:
They all should be total count.

Reply:
Then i still prefer exporting `totalRowCount` and use it here.

Reply:
null values of single index is in `NullCount` instead of in histogram.

Reply:
So I need to add `NullCount` to `count` for index result?

Reply:
Yes.